scipy/misc/__init__.py

```diff
@@ -8,33 +8,33 @@
 Various utilities that don't have another home.
 
 Note that the Python Imaging Library (PIL) is not a dependency
-of SciPy and therefore the `pilutil` module is not available on
-systems that don't have PIL installed.
+of SciPy, but the `pilutil` module and the image manipulation 
```

Comment:
While you're at it, can you remove ``the `pilutil` module``? It's not public so shouldn't be mentioned here.